committee-code-of-conduct/election.md

```diff
@@ -44,20 +45,25 @@ Nominations can be collected via any means. In the past a Google form has been u
 
 The nomination form must be communicated via email to [Kubernetes dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list.
 
-Prior to the voting process, Steering must:
-* Vet that all nominees support the role of the code of conduct committee, i.e. maintaining a safe and inclusive space for everyone to work on Kubernetes.
+Prior to the voting process, Steering Committee must:
+
+* Vet that all nominees support the role of the Code of Conduct Committee i.e., maintaining a safe and inclusive space for everyone to work on Kubernetes.
 * Confirm with all nominees (that were not self nominations) that they accept the nomination.
 
 ### Maximal representation
 
 Maximal representation of a single employer is `QUORUM - 1`. With a five person committee,
 that limits the number of members employed by the same company to two.
 
+### Committee Conflict of Interest
```

Comment:
Can we also include a similar note about Conflict of Interest in the steering election docs?

Reply:
Added in https://github.com/kubernetes/steering/pull/272!